packages/feedback-react/src/questions/RadioQuestion.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { ChangeEventHandler, useMemo } from "react";
+import { RadioButtons } from "@fremtind/jkl-radio-button-react";
+import { QuestionProps } from "../types";
+import { useFollowUpContext } from "../followup/followupContext";
+import { useMainQuestionContext } from "../main-question/mainQuestionContext";
+
+export const RadioQuestion: React.VFC<QuestionProps> = ({ label, name, options, helpLabel }) => {
+    const followupContext = useFollowUpContext();
+    const feedbackContext = useMainQuestionContext();
+    const context = followupContext || feedbackContext;
+
+    const handleChange: ChangeEventHandler<HTMLInputElement> = (e) => {
+        const option = options?.find((option) => option.value.toString() === e.target.value);
+        context?.setCurrentValue(option);
+    };
+
+    const selectedValue = useMemo(
+        () =>
+            Array.isArray(context?.currentValue)
+                ? context?.currentValue[0].value.toString()
+                : context?.currentValue?.value.toString(),
+        [context?.currentValue],
+    );
+
+    if (!context) {
+        console.error("Questions must be used inside a Followup or Feedback context provider");
+        return null;
+    }
+
+    return (
+        <RadioButtons
+            variant="large"
+            legend={label}
+            name={name || label}
```

Comment:
Dette gjelder kanskje `RadioButtons`, men VoiceOver leser opp både `legend` og `name` så jeg får "Fant du det du lette etter?" lest opp to ganger.

Reply:
Vet ikke helt om jeg klarer å gjenskape dette hos meg? På min maskin leser den bare opp valget når jeg bytter mellom knappene. Den leser opp "Fant du..." to ganger når man går inn i `fieldset`et, men alt det kommer fra `legend`: `name` blir bare brukt på selve radioknappene :)

Reply:
Ja, sorry, litt upresis. Den doble lesinga kommer bare én gang som du skriver, når jeg går inn i fieldsettet.